.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
+}
```

Comment:
You should have a new line at the end of all your files.

Reply:
Okay